### PR TITLE
[DowngradePhp74] Skip php doc method on DowngradeContravariantArgumentTypeRector

### PR DESCRIPTION
**Pull request by samsonasik**
Close #718 Fixes https://github.com/rectorphp/rector/issues/6651

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: rules-tests/DowngradePhp74/Rector/ClassMethod/DowngradeContravariantArgumentTypeRector/Fixture/skip_phpdoc_method.php.inc [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?php
```
Comment (Bl00D4NGEL) [inline rules-tests/DowngradePhp74/Rector/ClassMethod/DowngradeContravariantArgumentTypeRector/Fixture/skip_phpdoc_method.php.inc:1]: Isn't this the exact same? You can probably remove the "expectation" below the -----

Comment (samsonasik) [reply rules-tests/DowngradePhp74/Rector/ClassMethod/DowngradeContravariantArgumentTypeRector/Fixture/skip_phpdoc_method.php.inc:1]: fixed.

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**